pyproject.toml

```diff
@@ -13,7 +13,10 @@ classifiers = [
     "Programming Language :: Python :: 3.11",
 ]
 description = "One line description of your module"
-dependencies = [] # Add project dependencies here, e.g. ["click", "numpy"]
+dependencies = [
+    "numpy",
+    "click",
+] # Add project dependencies here, e.g. ["click", "numpy"]
```

Comment:
Can you put this back to blank please

Reply:
Oops, must've accidentally checked in `pyproject.toml`